### PR TITLE
[padctrl] D0 -> D1 Review

### PR DESCRIPTION
**Pull request by msfschaffner**
This proposes to move the padctrl to D1. I'll add the reviewer names once it is clear who signs this off.

Signed-off-by: Michael Schaffner msf@google.com

### CODE REVIEW

**Comment by msfschaffner** [comment]
Carried over some corrections from #928

File: hw/ip/padctrl/doc/checklist.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+[DESIGN_SPEC_REVIEWED]:               {{<relref "/doc/project/checklist.md#design-spec-reviewed" >}}
+[DV_PLAN_TESTPLAN_REVIEWED]:          {{<relref "/doc/project/checklist.md#dv-plan-testplan-reviewed" >}}
+[STD_TEST_CATEGORIES_PLANNED]:        {{<relref "/doc/project/checklist.md#std-test-categories-planned" >}}
+[V2_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v2-checklist-scoped" >}}
+
+### Checklists for milestone V2
+
+ Type         | Item                                    | Resolution  | Note/Collaterals
+--------------|-----------------------------------------|-------------|------------------
+Documentation | [DESIGN_DELTAS_CAPTURED][]              | Not Started |
```
Comment (sjgitty) [inline hw/ip/padctrl/doc/checklist.md]: name changed here in #912, please revise this to correspond.

File: hw/ip/padctrl/doc/checklist.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+[NO_HIGH_PRIORITY_ISSUES_PENDING]:    {{<relref "/doc/project/checklist.md#no-high-priority-issues-pending" >}}
+[ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED]:{{<relref "/doc/project/checklist.md#all-low-priority-issues-root-caused" >}}
+[PRE_VERIFIED_SUB_MODULES_V2]:        {{<relref "/doc/project/checklist.md#pre-verified-sub-modules-v2" >}}
+[V3_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v3-checklist-scoped" >}}
+
+### Checklists for milestone V3
+
+ Type         | Item                              | Resolution  | Note/Collaterals
+--------------|-----------------------------------|-------------|------------------
+Documentation | [DESIGN_DELTAS_CAPTURED_IF_ANY][] | Not Started |
```
Comment (sjgitty) [inline hw/ip/padctrl/doc/checklist.md]: here too

**Comment by msfschaffner** [comment]
Thanks, amended.

**Pull request merged**